README.md

```diff
@@ -18,6 +18,8 @@ is the recommended location for guidance on using this provider.
 
 ## Installation
 
+**NOTE**: This provider is built with the assumption that Buildkite Teams are enabled for your organisation. Most resources should work without, but we can't guarantee compatibility. Check out our [documentation regarding Teams](https://buildkite.com/docs/pipelines/permissions#permissions-with-teams) for more information.
```

Comment:
I think generally we write publicly with American spelling (I know, but  ¯\_(ツ)_/¯) and I think I'd use little-t teams, so would gently suggest:

```suggestion
**NOTE**: This provider is built with the assumption that teams are enabled for your Buildkite organization. Most resources should work without, but we can't guarantee compatibility. Check out our [documentation regarding Teams](https://buildkite.com/docs/pipelines/permissions#permissions-with-teams) for more information.
```

But otherwise this sounds great!